hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.controller.js

```diff
@@ -316,6 +313,16 @@ define([
         $scope.entity.leave[index].add_public_holidays = utils.absenceTypes[absenceTypeID].add_public_holiday_to_entitlement;
       });
     }
+
+    /*
+     * Set the entity object
```

Comment:
it's not though, is it? First of all it's copy the entity into the model (so it's doing more things that it's saying), and then it's not setting the `entity` object, it's setting the `contract` property of `entity`

Reply:
Removed the function all together.